src/subscription_manager/managercli.py

```diff
@@ -973,6 +974,89 @@ def list_service_levels(self):
                 raise e
 
 
+class UsageCommand(OrgCommand):
+
+    def __init__(self):
+
+        shortdesc = _("Manage usage setting for this system")
+        self._org_help_text = _("specify an organization when listing available service levels using the organization key, only used with --list")
+        super(UsageCommand, self).__init__("usage", shortdesc,
+                                                  False)
+
+        self._add_url_options()
+        self.parser.add_option("--show", dest="show", action='store_true',
+                help=_("show this system's current usage setting"))
+        self.parser.add_option("--set", dest="usage",
+                               help=_("usage setting to apply to this system"))
```

Comment:
Can you make the indentation consistent in this section?  I.e. every "help=" is at the same column.  I think PyCharm's default behavior is to indent everything to the first parenthesis in the line above which is terrible since it wastes so much space and everything ends up looking ragged.  I'd like to start just indenting everything one level in.  I've got it working in my set up.  Remind me and I'll show you whatever it was I did.